test/functional/JavaAgentTest/playlist.xml

```diff
@@ -46,7 +46,7 @@
 			<group>functional</group>
 		</groups>
 		<subsets>
-			<subset>9+</subset>
+			<subset>9|10|11</subset>
```

Comment:
Does '|' delimited actually work (get parsed properly by TKG)?  As 9/10 are now retired, I guess you could also just use 11 here if not.

Reply:
Thanks @smlambert , from a local test, `make -f run_configure.mk` and `make compile` finished successfully and `make _TestFlushReflectionCache_0` didn't run the test in question (was able to reproduce the failure `java.lang.NoSuchFieldException: annotations`.
Will remove `10` then.

Reply:
@smlambert PR has been updated, please review again.